client/src/pages/ForumPage/ForumPageDiscover.js

```diff
@@ -20,7 +23,32 @@ const handleClick = e => {
   console.log("Clicked");
 };
 
-const ForumPageDiscover = props => {
+
+class ForumPageDiscover extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      hashtags: [],
+    }
+  }
+
+  componentDidMount() {
+  axios.get(`${BASE_URL}/hashtags`)
+    .then(response => {
+      this.setState({ 
+        hashtags: response.data
+      })
+      console.log(response.data);
+    })
+    .catch(error => {
+      console.log('Error', error);
+    })
+}
```

Comment:
Can you put this in the appropriate service in the `services/` directory?

Reply:
Added Forum.js page in services/ directory and created import/export of axios fetch